app/libs/js/LEAF/formGrid.js

```diff
@@ -861,20 +840,40 @@ var LeafFormGrid = function (containerID, options) {
   }
 
   /**
-   * Set the working data set
+   * Set the working data set (uncommon)
+   * 
+   * setDataBlob() is more commonly used
+   * 
    * @params array - Expects format: [{recordID}, ...]
    * @memberOf LeafFormGrid
    */
   function setData(data) {
+    usingSetData = true;
     isDataLoaded = true;
     currentData = data;
+    processedCallbackBuffer = false;
+    document.querySelector(`#${prefixID}table`).setAttribute('aria-rowcount', currentData.length);
   }
 
   /**
+   * Set the working data set
+   * @params object - If setData() is not used, this expects {id: {recordID, ...}}
    * @memberOf LeafFormGrid
    */
   function setDataBlob(data) {
     dataBlob = data;
+    if(!usingSetData) {
+      if(Array.isArray(data)) {
+        currentData = data;
+      }
+      else {
+        currentData = Object.keys(data).map(key => data[key]);
```

Comment:
This function looks identical the the one I commented on above.  Do they need to be pulled out an put in a different file for both to reference one source?

Reply:
The directory for this file is part of an incomplete migration